docs/changelog.md

```diff
@@ -98,7 +98,11 @@ permalink: /changelog/
 
 * **feature-syncedtabs**
   * ⚠️ **This is a breaking change**: When constructing a `SyncedTabsStorage`, the `tabsStorage: RemoteTabsStorage` parameter is no longer optional so must be supplied [#11799](https://github.com/mozilla-mobile/android-components/pull/11799).
-
+  
+* **lib-auth**
+  * Add `lib-auth` for authentication using biometrics or PIN.
+    [issue # 12289](https://github.com/mozilla-mobile/android-components/issues/12289)
```

Comment:
This is in the wrong section. We're at 103 now.

Reply:
Done